_clients/java-rest-high-level.md

```diff
@@ -1,28 +1,29 @@
 ---
 layout: default
-title: Java high-level REST client
-nav_order: 97
+title: OpenSearch Java high-level REST client
+nav_order: 60
 ---
 
-# Java high-level REST client
+# OpenSearch Java high-level REST client
 
-The Elasticsearch OSS Java high-level REST client allows you to interact with your OpenSearch clusters and indices through Java methods and data structures rather than HTTP methods and JSON.
+Although the OpenSearch Java high-level REST client is still usable, we recommend that you use the [OpenSearch Java client]({{site.url}}{{site.baseurl}}/clients/java/), which replaces the existing Java high-level REST client.
+{: .note}
 
-You submit requests to your cluster using request objects, which allows you to create indices, add data to documents, or complete other operations with your cluster. In return, you get back response objects that have all of the available information, such as the associated index or ID, from your cluster.
+The OpenSearch Java high-level REST client lets you interact with your OpenSearch clusters and indices through Java methods and data structures rather than HTTP methods and JSON.
 
 ## Setup
 
-To start using the Elasticsearch OSS Java high-level REST client, ensure that you have the following dependency in your project's `pom.xml` file:
+To start using the OpenSearch Java high-level REST client, ensure that you have the following dependency in your project's `pom.xml` file:
 
 ```
 <dependency>
-  <groupId>org.elasticsearch.client</groupId>
-  <artifactId>elasticsearch-rest-high-level-client</artifactId>
-  <version>7.10.2</version>
+  <groupId>org.opensearch.client</groupId>
+  <artifactId>opensearch-rest-high-level-client</artifactId>
+  <version>1.1.0</version>
```

Comment:
Any disadvantage to making this a variable? AKA `{{site.opensearch_version}}`?

Reply:
Good call, I will add this in.